container-dependency-versions/pom.xml

```diff
@@ -383,6 +382,11 @@
                 <artifactId>xml-apis</artifactId>
                 <version>1.4.01</version>
             </dependency>
+            <dependency>
```

Comment:
There are now two 'dependency' entries for lz4 in this pom file. Keep just one, and please preserve lexical sorting on groupId.

Reply:
Fixed